<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nextreport</class>
 <widget class="QWidget" name="nextreport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="necessaryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Is it neccessary?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nextEntrylabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next Entry</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectedVehicle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vehicleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vehicle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="maintenanceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maintenance Type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="complete">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Complete</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectedNecessary">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="selectedMaintenance">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="notesBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>complete</sender>
   <signal>clicked()</signal>
   <receiver>nextreport</receiver>
   <slot>complete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>nextreport</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>complete()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
